tests/units/array/test_mixins/test_attribute.py

```diff
@@ -0,0 +1,168 @@
+from typing import Optional, Union
```

Comment:
we don't have mixin in array anymore. we should move this file to somewhere else

Reply:
did u write all of these tests? I don't understand how they are all relevant.

Reply:
moved to `test_array`